src/Aspire.Hosting.AWS/Utils/ProjectUtilities.cs

```diff
@@ -0,0 +1,123 @@
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Aspire.Hosting.AWS.Utils;
+
+internal static class ProjectUtilities
+{
+    /// <summary>
+    /// Initializes the project's launch settings if necessary, and
+    /// ensures they are referencing the Amazon.Lambda.TestTool's location.
+    /// </summary>
```

Comment:
can you add docs for each param. and maybe a link/sample of the generated profile?